pkg/cip/audit/auditor_test.go

```diff
@@ -0,0 +1,1017 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package audit_test
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"reflect"
+	"regexp"
+	"testing"
+
+	"k8s.io/release/pkg/cip/audit"
+	reg "k8s.io/release/pkg/cip/dockerregistry"
+	"k8s.io/release/pkg/cip/logclient"
+	"k8s.io/release/pkg/cip/remotemanifest"
+	"k8s.io/release/pkg/cip/report"
+	"k8s.io/release/pkg/cip/stream"
+)
+
+func checkMatch(haystack []byte, re *regexp.Regexp) error {
+	if !re.Match(haystack) {
+		return fmt.Errorf(
+			"===BEGIN-MESSAGE===\nCOULD NOT FIND MATCH FOR %q IN\n%s\n===END-MESSAGE===",
+			re.String(),
+			haystack)
+	}
+	return nil
+}
+
+func checkEqual(got, expected interface{}) error {
```

Comment:
we can remove this and use the testify package `"github.com/stretchr/testify/require"`  but we can do in a followup

Reply:
Exactly what I was thinking! I didn't want to touch it in this PR though.

Reply:
PR: https://github.com/kubernetes/release/pull/1772